keyboards/kyria/keymaps/plattfot/keymap.c

```diff
@@ -0,0 +1,437 @@
+/* Copyright 2020 Fredrik Salomonsson <plattfot@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+#include "version.h"
+
+enum custom_keycodes {
+    PLACEHOLDER = SAFE_RANGE,  // can always be here
+    // Macros, where | is the cursor
+    M_LRBRC,  // [|]
+    M_LRCBR,  // {|}
+    M_LRPRN,  // (|)
+    M_LRABR,  // <|>
+    M_DQUOT,  // '|'
+    // New keys
+    DBL_TAP,  // Repeat next key
+};
+
+// Double Tap feature based on post from drashna
+// https://www.reddit.com/r/olkb/comments/citkbx/double_key_press_modifier_qmkwould_work_like/ev9cue8/
+static bool double_tap_it = false;
+
+enum layers {
+    _DEFAULT = 0,
```

Comment:
This is more of a nit pick, but this starts off as 0, no need to explicitly set it.

```suggestion
    _DEFAULT,
```